app/(dashboard)/dashboard/pull-request.tsx

```diff
@@ -22,6 +22,7 @@ import {
 import { Button } from "@/components/ui/button";
 import { Checkbox } from "@/components/ui/checkbox";
 import { Input } from "@/components/ui/input";
+import { Select } from "@/components/ui/select";
```

Comment:
IMO this should be in a new Settings page instead of per PR, or next to the filters.

Reply:
something like cursor?
ability to provide api keys for each provider,
enable disable each model
ability to add model as string

and set one mode as default/or choose only one model among availabel.

<img width="720" alt="image" src="https://github.com/user-attachments/assets/bd66b7e0-220d-42e4-8533-60e9de7c9758">

more guidance on how the feature would like like from your perspective will help.

Reply:
also my perspective here was as a developer, it was easy way for me to compare test generation from different models when this option is in per item,
I was thinking of capability to set default model for account though

I can understand, as a user this might not be the thing one want to select or think about each time they generate the tests..

Reply:
Think that's not keeping in shortest config personally. Seems like part of the value is having shortest make those changes to always use SOTA

Reply:
@juniusfree in my opinion, shortest.yml should not have any repo specifications, current app lets us connect our github account and work with any repo, 
having repo config in yml seems like pulling away that functionality.
Let me know if I understood it wrong.

Shortest currently does not have repo listing page or per repo config ui, something like codeclimate's ui for per repo config would probably be better place to manage per repo config.
<img width="880" alt="image" src="https://github.com/user-attachments/assets/07caf047-8e36-44b7-8a00-a7af423a98fb">

I think for current use case, being ability to select preferred model for account would be enough.
If there were some initial settings page design, it would be easier for me to continue(I assume core-team would have highly opinionated designs for such new pages)